src/actions/evaluation-queue.js

```diff
@@ -6,9 +6,17 @@ export class EvaluationQueue {
     this.queueSize = 0;
     this.evaluationResolvers = {};
     this.idGenerator = new IdFactory();
+    this.reduxDispatch = null;
   }
 
-  evaluate(chunk, dispatch) {
+  connectToRedux(dispatch) {
```

Comment:
could this be `connectToStore`? "Redux" is kind of vague

Reply:
also, this is a great refactor in general

Reply:
i'll do `connectDispatch` to match the pattern from review comments #1428